src/jit/codegen.h

```diff
@@ -465,11 +465,11 @@ protected :
                                     IL_OFFSETX                                    ilOffset);
 
 
+#if defined(_TARGET_ARM_)
     //
```

Comment:
This appears to work. I'll go through the diffs and try to address as many of the ifdef moves as I can with this.